packages/appstore/src/modules/traders-hub/index.tsx

```diff
@@ -8,8 +8,8 @@ import AccountWithWallets from './account-with-wallets';
 import AccountWithoutWallets from './account-without-wallets';
 import { useStore, observer } from '@deriv/stores';
 import EUDisclaimer from 'Components/eu-disclaimer';
-import './traders-hub.scss';
 import { useContentFlag } from '@deriv/hooks';
+import './traders-hub.scss';
 
 const TradersHub = () => {
     const { traders_hub, client, ui } = useStore();
```

Comment:
Wrap the component with `observer()` when use `useStore()`

Reply:
@aizad-deriv observer uses in the end of component

`export default observer(TradersHub);`

I didn't change this behaviour in this component. I see sometimes we uses observer when export it, sometimes we wrap while component, I think there is no difference with these 2 ways.

Reply:
Can you update the file then to wrap the `observer` with the whole component.